src/main/git-elegant-feature

```diff
@@ -2,7 +2,17 @@
 set -e
 
 default() {
+    local uncommited_changes=false
     _validate "$1" "Feature name"
+    local status="git status | grep 'Changes to be commited: || Changes not staged for commit:'"
+    if [ -n $(eval "$status") ]; then
+        git stash
```

Comment:
We need to set some name for a stash:
```
git stash save "git-elegant: Save changes to unstash them to '$1' branch"
```

Reply:
@extsoft Do you want to use this stash name?
>git stash save "git-elegant: Save changes to unstash them to '$1' branch"

Maybe something shorter? For example:
```
git stash save "stash made by git-elegant"
```